tests/integration/frameworks/wordpress/test_wordpress_apply_filters.php8.php

```diff
@@ -36,6 +39,7 @@
     [{"name": "DurationByCaller/Unknown/Unknown/Unknown/Unknown/allOther"}, [1, "??", "??", "??", "??", "??"]],
     [{"name": "Framework/WordPress/Hook/f"},                          [1, "??", "??", "??", "??", "??"]],
     [{"name": "Framework/WordPress/Hook/g"},                          [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Framework/WordPress/Hook/h"},                          [1, "??", "??", "??", "??", "??"]],
```

Comment:
Previously, Hook/h metric wasn't generated because we were using an 'after' wrapper and h threw an exception, preventing our wrapper from being called. Now, we wrap with a 'before' wrapper.

Reply:
Thanks for this detail! This helps a lot!